components/mosart/src/riverroute/MOSART_physics_mod.F90

```diff
@@ -682,8 +682,8 @@ subroutine Euler
 
 ! check for negative channel storage
     if (negchan < -1.e-10) then
-       write(iulog,*) 'Error: Negative channel storage found! ',negchan
-       call shr_sys_abort('mosart: negative channel storage')
+       write(iulog,*) 'Warning: Negative channel storage found! ',negchan
```

Comment:
are you sure you want to print a warning?  Was that the old behaviour?

Reply:
Yes looks like it was.

Reply:
Another PR, #6568, resolved this aborting issue with a cleaner solution. Should we discuss and then discuss how to proceed with these two options?

Reply:
This one will have to go in for the 3.0.1 tag.  Then your other fix can be used for 3.1.0.